### ISSUE TITLE
fetchPackageMetaData error for first-mate-select-grammar

### ISSUE THREAD

**Issue by i-bsd** [soldair/first-mate-select-grammar#3]
Just started getting this error on my Gatsby default starter today (never had a problem before):

```
8033 silly fetchPackageMetaData error for first-mate-select-grammar@git+https://github.com/atom/first-mate-select-grammar.git#b7d54921d849e6108f968581bf31e41929fd992f Command failed: /usr/bin/git checkout b7d54921d849e6108f968581bf31e41929fd992f
```
Is this something to do with the update earlier today?

**Comment by soldair** [comment]
@jasonrudolph made a pr here and in the mean time added for of this package as a git dep. This seems to be an error cloning that git dependency.

if the package you depend on uses the version of this package from npm you wont see this error. @jasonrudolph let me know if i missed something here =)

**Comment by soldair** [comment]
it looks like the fixed version was released in highlights@3.1.4
if you update to latest you should be good to go
